fashion-messaging-app-ui/src/App.jsx

```diff
@@ -28,42 +25,58 @@ export default function App() {
   };
   const [photos, setPhotos] = useState([]);
   const [searchQuery, setSearchQuery] = useState("");
+  const [noResults, setNoResults] = useState(false);
+
+useEffect(() => {
+  setLoading(true);
+
+  const query = searchQuery || defaultQuery;
+  clientAPI.photos
+    .search({ query, per_page: PHOTOS })
+    .then(async response => { 
+      setLoading(false);
+
+      var myHeaders = new Headers();
+      myHeaders.append("x-api-key", "1a4092c4563cc701627b1616fa6053190c6b538fd82b828d3fe20416bcc3dd3e");
+
+      let fetchPromises = await Promise.all(response.photos.map(async photo => { 
+        var formdata = new FormData();
+        formdata.append("image_url", photo.src.original);
```

Comment:
I changed the previous url so that this will fetch a response using the url from the object that my first API returns